packages/core/src/components/Tooltip/index.tsx

```diff
@@ -58,7 +67,7 @@ export class Tooltip extends React.Component<Props & WithStylesProps, State> {
     labelID: uuid(),
     open: false,
     tooltipHeight: 0,
-    targetRect: document.body.getBoundingClientRect(),
+    targetRect: document.body ? document.body.getBoundingClientRect() : defaultTargetRect,
```

Comment:
I wonder if moving this to cDM would solve the problem.

Reply:
moved it!